docs/quickstart.rst

```diff
@@ -0,0 +1,55 @@
+
+.. _quickstart:
+
+Quickstart
+==========
+
+Once installed, ``astrogrism`` can be imported and used in an interactive Python
+session or in a Jupyter notebook. The main class used to read in grism
+observation FITS files and expose the functionality is the ``astrogrism.GrismObs`` 
+class.
+
+To load your data, simply run the following in a Python interpreter or Jupyter
+notebook cell::
+    
+    from astrogrism import GrismObs
+    g_obs = GrismObs("sample_grism_flt.fits")
+
+This object makes available the tranforms between the ``world``, ``detector`` 
+(i.e. the undispersed direct image), and ``grism_detector`` frames. To get 
+the transform between, for example, the world and grism detector frames, you 
+can call::
+
+    world_to_grism = g_obs.geometric_transforms.get_transform("world", "grism_detector")
+
+For instruments with two chips (WFC3 UVIS and ACS), you must specify the chip for
+which you want the transform, e.g.::
+   
+    world_to_grism = g_obs.geometric_transforms["CCD1"].get_transform("world", "grism_detector")
+
+This transform would then allow you to calculate the coordinates on the dispered image
+given a right ascension and declination (currently required to be in degrees), the
+wavelength of interest, and the order of the dispersed spectrum::
+
+    world_to_grism(264.10183, -32.90802, 0.7, 1.0)
+
+This returns a tuple of values (x_d, y_d, x_g, y_g, order), where _d denotes 
+coordinates on the direct image, and _g denotes the dispersed coordinates on 
```

Comment:
Pardon if I got the order wrong, but I think it would help if we mirror the labels we use in our docstrings: 
https://github.com/spacetelescope/astrogrism/blob/main/astrogrism/HST/transform_models.py#L60-L73

```suggestion
This returns a tuple of values (x, y, x0, y0, order), where (x, y) denotes 
coordinates on the direct image, and (x0, y0) denotes the dispersed coordinates on 
```

Reply:
Good call. I think this might apply one other place, I'll double check.